cmd/habitat/community/state/transitions.go

```diff
@@ -211,3 +223,158 @@ func (t *AddNodeTransition) Validate(oldState *community.CommunityState) error {
 
 	return nil
 }
+
+type StartProcessTransition struct {
+	Process *community.Process
+}
+
+func (t *StartProcessTransition) Type() string {
+	return TransitionTypeStartProcess
+}
+
+func (t *StartProcessTransition) Patch(oldState *community.CommunityState) ([]byte, error) {
+	marshaledProcess, err := json.Marshal(t.Process)
+	if err != nil {
+		return nil, err
+	}
+
+	return []byte(fmt.Sprintf(`[{
+		"op": "add",
+		"path": "/processes/-",
```

Comment:
what does the `-` here mean?

Reply:
It tells the JSON patch to append the new process to the end of the array